analysis/organize_data.qmd

```diff
@@ -5,16 +5,16 @@ title: "Organize Data Notebook"
 ```{r}
 #| label: setup
 #| include: false
-library(tidyverse)
-library(lubridate)
-library(tidytext)
+library(here)
```

Comment:
Don't add new packages here. Add them to `check_packages.R`. Also you need the `here` package or you won't be able to source the files in the next two lines.